easybuild/easyconfigs/c/CASA/CASA-6.4.4.eb

```diff
@@ -0,0 +1,29 @@
+# Contribution from Imperial College London
+# Uploaded by J. Sassmannshausen
+
+easyblock = 'Tarball'
+
+name = 'CASA'
+version = '6.4.4'
+
+homepage = 'https://casa.nrao.edu'
+description = """CASA, the Common Astronomy Software Applications package, is the 
+primary data processing software for the Atacama Large Millimeter/submillimeter 
+Array (ALMA) and NSF's Karl G. Jansky Very Large Array (VLA), and is frequently 
+used also for other radio telescopes. The CASA software can process data from 
+both single-dish and aperture-synthesis telescopes, and one of its core 
+functionalities is to support the data reduction and imaging pipelines for 
+ALMA, VLA and the VLA Sky Survey (VLASS)."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://alma-dl.mtk.nao.ac.jp/ftp/casa/distro/casa/release/rhel']
+sources = ['casa-%(version)s-31-py3.8.tar.xz']
+checksums = ['c66fb031d2d7b937198ba4050fa882ea09e9e8c13a289e60e70a8cb0e80fb1d3']
+
+sanity_check_paths = {
+    'files': ['bin/casa', 'bin/mpicasa'],
```

Comment:
`mpicasa`? That does not look promising...

Reply:
Both are Perl scripts setting up the environment. At least they seem to check a few things as far as I can see. 
So in the end, it seems to call the `mpirun` it found from the environment.

Reply:
But if you don't depend on an MPI implementation then there is no `mpirun` in the environment

Reply:
I had another look at it and it appears that this is installing its own environment, i.e. `MPI` and `Python` next to quite a bit more. I mark that as WIP so I can have a closer look at it. Sorry for overlooking that in the first instance.